ionic_user_interface/src/components/GymRouteList.vue

```diff
@@ -108,10 +108,7 @@
         </ion-card-header>
         <ion-card-content>
           <b>Creator:&nbsp;</b>
-          <router-link
-            :to="'/userRoutes/' + route.username"
-            @click.capture.prevent.stop="undefined"
-          >
+          <router-link :to="'/userRoutes/' + route.username" @click.capture.stop="undefined">
```

Comment:
```suggestion
          <router-link :to="'/userRoutes/' + route.username" @click.capture.stop="() => undefined">
```

To get rid of a console warning when clicking